### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by donnelldebnam**
The branch is names about_page, but I also made changes to the homepage. I took out all of our personal info and refactored it into the "About Us" page. 

I included 'awesome' icons that will link to our different social media pages, email addresses, etc. I encourage everyone to set the links to their correct information and edit the short bio. (I had to use some placeholders because I couldn't find the information myself)

### CODE REVIEW

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -22,38 +22,117 @@
...
+               </div>
+                 <!-- Jaiyi Wang-->
+               <div class="col-xxs-12 col-sm-6 col-xxl-3 m-b-xxl">
+                <br>
+                 <img class="img-circle img-fluid m-x-auto m-b" style="width: 112px;"
+                      src="http://i1067.photobucket.com/albums/u437/donnedebnam1/imageedit_12_8988254368_zpssfyxzbsx.gif">
+                 <h3 class="text-md text-bold">Jaiyi Wang</h3>
+                 <p class="text-xxxs m-b-sm text-uppercase">Virginia Tech</p>
+                 <p class="text-xs m-b-sm m-x-auto" style="max-width: 35rem;">
+                 Jaiyi is a driven, third year, Computer Science student at Virginia Tech in Blacksburg, VA!
```
Comment (jiayiw6) [inline src/main/webapp/about.jsp]: It's Jiayi XD

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 2]
```diff
@@ -94,7 +94,7 @@ List<String> hashWords = new ArrayList<String>();
         <form action="/users/<%=request.getSession().getAttribute("user") %>" method="POST">
           <div class="form-group">
             <label class="form-control-label">Edit Your About Me:</label>
-            <textarea rows="5" cols="75" name="About Me" placeholder="I'm currently a student at..."></textarea>
+            <textarea class="form-control"rows="5" cols="75" name="About Me" placeholder="I'm currently a student at..."></textarea>
```
Comment (jiayiw6) [inline src/main/webapp/WEB-INF/view/profile.jsp]: What does class=form-control actually do?

File: src/main/webapp/about.jsp (hunk 1 above)
Comment (donnelldebnam) [reply src/main/webapp/about.jsp]: Soooo sorry!

File: src/main/webapp/WEB-INF/view/profile.jsp (hunk 2 above)
Comment (donnelldebnam) [reply src/main/webapp/WEB-INF/view/profile.jsp]: it pretty much just gives the textarea or input area a 100% width and can style the area. In this case, it aligned the textarea box up with the "Sent Messages" and gave the textarea a more rounded and cooler look.

**Pull request merged**